dev/test_unit

```diff
@@ -10,6 +12,7 @@ docker run \
     -e CONJUR_AUTHN_LOGIN=admin \
     -e SECURITY_USER_NAME=TEST_USER_NAME \
     -e SECURITY_USER_PASSWORD=TEST_USER_PASSWORD \
+    -v "$(pwd)"/..:/app \
```

Comment:
Out of curiosity, is there any benefit or drawbacks from using this, over having `cd "$(dirname "$0")/.."` at the top of the file, and mounting `$(pwd)` directly here?

Reply:
I'm thinking that tests scripts for a given repo will likely share code/utils, so maybe it's good to default them to running out of the directory that they're in?

Reply:
Good point!